lib/Transport.js

```diff
@@ -963,8 +965,8 @@ class Transport extends Emitter
 	{
 		//Create new incoming stream
 		let outgoingStream = this.createOutgoingStream ({
-			audio: incomingStream.getAudioTracks().length,
-			video: incomingStream.getVideoTracks().length
+			audio: incomingStream.getAudioTracks().map(t => ({})),
+			video: incomingStream.getVideoTracks().map(t => ({})),
```

Comment:
shouldn't we just change the typescript definition instead?

```
/**
 * @typedef {Object} CreateStreamOptions
 * @property {string} [id]
 * @property {Array<CreateStreamTrackOptions>|CreateStreamTrackOptions | Number} [audio] Add audio track to the new stream
 * @property {Array<CreateStreamTrackOptions>|CreateStreamTrackOptions | Number} [video] Add video track to the new stream
 */
```

Reply:
no, the typescript definition correctly reflects what the function accepts.
if you want, what we can do is change the code to accept a single number

Reply:
Yes, somehow the piece of code using Number as input was never removed or never implemented..  

As it feels a little weird to have an array of empty objects, how about keeping the track id on the new stream? 

```suggestion
			video: incomingStream.getVideoTracks().map(t => ({ id : t.id })),
```

Reply:
looks like a good idea. the suggestion is missing the audio tracks, so I'll apply the change manually